src/components/quiz/QuizConstructor.vue

```diff
@@ -31,68 +34,46 @@ defineProps({
 
 const selectedCategory = ref();
 const checkedQuestions = ref([]);
+const questionList = ref<QuizQuestion[]>([]);
 const checkedAnswer = ref(0);
 const currentCandidateId = ref(0);
+const currentCandidateName = ref("");
 const startQuizDate = ref(0);
+const resultsStore = useResultsStore();
```

Comment:
lets keep hooks before refs and add an empty line between to separate them and add a bit of readability